plugins/web/opentelemetry-instrumentation-document-load/test/documentLoad.test.ts

```diff
@@ -535,14 +538,16 @@ describe('DocumentLoad Instrumentation', () => {
         assert.strictEqual(fetchSpan.name, 'documentFetch');
         assert.strictEqual(rootSpan.name, 'documentLoad');
 
-        assert.strictEqual(
-          fetchSpan.attributes['http.url'],
-          'http://localhost:9876/context.html'
+        assert.isOk(
+          (fetchSpan.attributes['http.url'] as string).startsWith(
+            'http://localhost:8000/?wtr-session-id='
```

Comment:
URL is different, and the session ID unique per run. I think this is fine, but happy to change to whatever you want 🙂